docs/edgeql/insert.rst

```diff
@@ -117,6 +117,25 @@ You can only ``insert`` instances of concrete (non-abstract) object types.
   ... };
   error: QueryError: cannot insert into abstract object type 'default::Person'
 
+By default, ``insert`` returns only the inserted object's ``id`` as seen in the
+examples above. If you want to get additional data back, you may wrap your
+``insert`` with a ``select`` and apply a shape specifying any properties and
+links you want returned:
+
+.. code-block:: edgeql-repl
+
+  db> select (insert Hero {
```

Comment:
Let's also offer an example of a `with` style, which I personally prefer as a better-looking formulation:

```
with
   NewHero := (insert)
select
   NewHero {
      id,
      name,
   }
```